lib/eex/test/eex_test.exs

```diff
@@ -175,7 +175,7 @@ defmodule EExTest do
     end
 
     test "when end expression has a modifier" do
-      assert_raise EEx.SyntaxError, ~s[nofile:1: unexpected beginning of EEx tag "<%=" on end of expression "<%= end %>", please remove "=" accordingly], fn ->
+      assert ExUnit.CaptureIO.capture_io :stderr, fn ->
```

Comment:
Not sure what you mean. The old test asserted on the exception message and this test needs to assert on the warning message. What do you mean that the message is not checked?